app/src/main/java/org/fossasia/openevent/app/common/data/AuthModel.java

```diff
@@ -132,4 +133,17 @@ public Completable submitToken(SubmitToken tokenData) {
             .subscribeOn(Schedulers.io())
             .observeOn(AndroidSchedulers.mainThread());
     }
+
+    @Override
+    public Completable changePasswordR(ChangePassword chPassword) {
```

Comment:
Please correct the spelling here -> `changePasswordR` 
Also expand the word completely ->`chPassword`